apps/k8s-io/certificate-canary.yaml

```diff
@@ -47,6 +47,8 @@ spec:
   - issue.kubernetes.io
   - issues.k8s.io
   - issues.kubernetes.io
+  - kep.k8s.io
+  - kep.kubernetes.io
```

Comment:
@enj This is a self-sign certificate we plan to deprecate.
```suggestion
```

Reply:
Okay, I have reverted the changes to this file.